designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DMScenarioCommand.scala

```diff
@@ -86,3 +80,9 @@ case class DMCancelScenarioCommand(scenarioName: ProcessName, user: User) extend
 
 case class DMStopScenarioCommand(scenarioName: ProcessName, savepointDir: Option[String], user: User)
     extends DMScenarioCommand[SavepointResult]
+
+case class DMPerformSingleExecutionCommand(
```

Comment:
1. What about custom actions? Can you write in the description the plan of changes that will be made in further PRs? 
2. Let's write a TODO describing further plan that won't be a part of currently planned changes. Do you agree that eventually we should extract some kind of a scheduler interface and in the deployment manager keep only logic related with the deployment? Or maybe you see it differently?

Reply:
I added in this PR description the general plan for periodics, especially the change that is already in progress. 

I'm not sure ATM about the future of CustomAction's. We aren't using them internally anymore - "run now" was our only usage. They are left in the codebased and APIs right now, so Nu users can still use them. 

I guess we need to decide, whether to leave CustomAction's as they are, or remove them.

Reply:
Custom action removal added to the plan.